docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,134 @@
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+<a name="create_instance"></a>
+## Creating an Widget Control Instance
+
+To create an instance on the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code snippet:
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
+<a name="getting_information"></a>
+## Getting Information on Widget Applications
+
+To get information on the widget applications, as shown in the following code snippet:
+
+    ```
+    string mainId = control.MainAppId;
+    string packageId = control.PackageId;
+    IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+    ```
+
+<a name="listening_events"></a>
+## Listening Widget Lifecycle Events on Widget Applications
+
+To listen to the widget lifecycle events, add lifecycle listeners on the control, as shown in the following code snippet:
+
+    ```
+    private static void OnCreated(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        string instanceId = args.InstanceId;
+        string widgetId = args.WidgetId;
+        /// Widget application is created
+    }
+
+    private static void OnDestroyed(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        /// Widget application is destroyed
+    }
+
+    private static void OnPaused(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        /// Widget application is paused
+    }
+
+    private static void OnResumed(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+    {
+        /// Widget application is resumed
+    }
+
+    control.Created += OnCreated;
+    control.Created += OnDestroyed;
+    control.Created += OnPaused;
+    control.Created += OnResumed;
+
+    string mainId = control.MainAppId;
+    string packageId = control.PackageId;
+    IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+        ```
+
+<a name="communicating_instances"></a>
+## Communicating with Running Widget Instances
+
+To communicate with the running widget instances, follow these steps:
+
+1.  Operate on the control:
+    -   Get running widget instances:
+
+        ```
+        IEnumerable<WidgetControl.Instance> instances = control.GetInstances();
+        ```
+
+2.  Operate on the instances:
+    -   Get details of running widget instances and send an update to widget application:
+
+        ```
+        foreach (WidgetControl.Instance ins in instances) {
+            /// Get widget instance content
+            var data = ins.GetContent();
+
+            /// Change widget update period
+            ins.ChangePeriod(2.0);
+
+            /// Trigger widget update
+            ins.ChangeContent(data, true);
+        }
+        ```
```

Comment:
Question to @gotoextreme. Don't we have a way of "Destroying Widget Control Instance"?

Reply:
The widget control class inherited IDisposable and it is obvious that a developer can destroy it using Dispose() function. I think it should be mentioned on the Dispose class document.